src/code/main.c

```diff
@@ -50,15 +50,15 @@ void Main(void* arg) {
     PreNmiBuff_Init(gAppNmiBufferPtr);
     Fault_Init();
     SysCfb_Init(0);
-    sysHeap = (u32)gSystemHeap;
+    sysHeap = gSystemHeap;
```

Comment:
Afaict `uintptr_t` is defined with `void*` pointers so does casting to `uintptr_t` first requires a cast to `void*`? Like:
```suggestion
    sysHeap = (void*)gSystemHeap;
```

Reply:
I dont think it needs it since that variable is an array. If anything it should have a `uintptr_t` cast.

Reply:
My point was `gSystemHeap` is a `u8*`, not a `void*`, and `uintptr_t` is for `void*` pointers afaict